.env

```diff
@@ -0,0 +1,2 @@
+REST_ENV=mock
```

Comment:
You should ignore this file and does not upload to repository, it's an environment that we would like to replace in production, development, etc.